File: Python/List-2/sum13.py
# my solution
def sum13(nums):
	n = len(nums)
	if n == 0:
		return 0
	for i in range(n):
		if nums[i] == 13:
			nums[i] = 0
			if i + 1 < n:
				nums[i + 1] = 0

	return sum(nums)

File: Java/Array-2/sum13.java
public int sum13(int[] nums) {
  int total = 0;
  int len = nums.length;
  // arr is empty
  if (len == 0)
    return 0;
    
  int i = 0;
  while (i < len)
  {
    if (nums[i] == 13)
      i += 2;
    else
    {
      total += nums[i];
      i++;
    }
  }
  return total;
}
